src/gl/tessellation_helpers.cpp

```diff
@@ -14,32 +14,58 @@
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 #include "mir/gl/tessellation_helpers.h"
+#include "mir/geometry/forward.h"
```

Comment:
It seems odd that we'd want `forward.h` in an implementation file

Reply:
Oops, we don't, but `clangd` always wants to pull it in. I missed this one!